bin/check_drupal

```diff
@@ -293,6 +293,11 @@ check_security_updates() {
 		return $EXIT_UNKNOWN
 	fi
 
+	# Remove via drush locked updates, keep the rest
```

Comment:
Because there are two different functions? check_security_updates() and check_system_updates()